test/docs.js

```diff
@@ -805,20 +878,29 @@ describe('Docs', function() {
 
   it('should be able to refresh the lock with the same tabId', async function() {
     const req = apos.task.getReq();
-    const doc = await apos.doc.db.findOne({ _id: 'i27:en:published' });
+    await insertPeople(apos);
+    const doc = await apos.doc.db.findOne({ _id: 'carl:en:published' });
+    const wait = (time) => new Promise((resolve) => setTimeout(() => {
+      resolve();
+    }, time));
 
     try {
       await apos.doc.lock(req, doc, 'abc');
+      const locked = await apos.doc.db.findOne({ _id: 'carl:en:published' });
+      await wait(500);
```

Comment:
I needed to wait here otherwise the test is flaky. If you lock again with the same `updatedAt` the update isn't considered because the data is the same and the test fail.